koku/masu/api/sourceproviders/sources/serializers.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2021 Red Hat Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Serializers for Masu API `manifest`."""
+from rest_framework import serializers
+
+from api.provider.models import Sources
+
+
+class SourceSerializer(serializers.Serializer):
+    """Serializer for Soruces."""
+
+    class Meta:
+        model = Sources
+
+    source_id = serializers.IntegerField()
+    source_uuid = serializers.UUIDField()
+    name = serializers.CharField()
+    auth_header = serializers.CharField()
+    offset = serializers.IntegerField()
+    account_id = serializers.IntegerField()
```

Comment:
In the Sources model, what is the type of `account_id`? Why is this field an `IntegerField`?

Reply:
This should be a char field this has been changed.